rhel8/profiles/cis.profile

```diff
@@ -601,87 +601,88 @@ selections:
 
     ### 4.1.9 Ensure discretionary access control permission modification
     ###       events are collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5509
     
     ### 4.1.10 Ensure unsuccessful unauthorized file access attempts are
     ###        collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5510
 
     ### 4.1.11 Ensure events that modify user/group information are
     ###        collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5511
 
     ### 4.1.12 Ensure successful file system mounts are collected (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5512
```

Comment:
I think we could cover it with audit_rules_media_export

Reply:
Likely! Feel free to self-assign that ticket and review/patch as applicable.